prac_07/guitar_file_reader.py

```diff
@@ -0,0 +1,34 @@
+from guitar import Guitar
+
+GUITAR_FILENAME = 'guitars.csv'
+
+
+def main():
+    guitars = []
+    in_file = open('guitars.csv', 'r')
+    in_file.readline()
+    for line in in_file:
+        parts = line.strip().split(',')
+        name = parts[0]
+        year = int(parts[1])
+        cost = float(parts[2])
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(guitar_to_add)
```

Comment:
Could you extract this as a separate function?
Also, `gutiar_to_add` kind of looks like it should be a dictionary